web/src/containers/students/Student.css

```diff
@@ -0,0 +1,109 @@
+
```

Comment:
Is this file used? If not delete it and clean html code

Reply:
era antigo esqueci-me de o eliminar no merge...

Reply:
tens que eliminar as classes do html também

Reply:
deixei só para ter uma noção do que cada coisa faz, acho que nao faz grande diferença no resultado final mas posso apagar